### PR TITLE
creation pages home and caroussel

### PR DESCRIPTION
**Pull request by n-faycel**

### CODE REVIEW

**Review by ThomasDCW — DISMISSED** [review]
ok

**Review by LuigiMenez — APPROVED** [review]
Sorry I didn't see the conflicts

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Fix conflict.

**Review by ThomasDCW — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

File: frontend/src/pages/Home/index.jsx [hunk 1]
```diff
@@ -1,15 +1,54 @@
-import Header from "@components/Header";
-import Navbar from "@components/Navbar";
+// eslint-disable-next-line import/no-unresolved
+import { Swiper, SwiperSlide } from "swiper/react";
+// eslint-disable-next-line import/no-unresolved
+import "swiper/css";
+// eslint-disable-next-line import/no-unresolved
+import "swiper/css/navigation";
+// eslint-disable-next-line import/no-unresolved
+import "swiper/css/pagination";
```
Comment (LoicBrassart) [inline frontend/src/pages/Home/index.jsx:8]: Sorry, I should have been more explicit for this one :pray: 
```
// eslint-disable import/no-unresolved
import { Swiper, SwiperSlide } from "swiper/react";
import "swiper/css";
import "swiper/css/navigation";
import "swiper/css/pagination";
import { Navigation, Pagination, Mousewheel, Keyboard } from "swiper";
// eslint-enable import/no-unresolved
```

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
One fix and it's my "fault", sorry for that

**Pull request closed without merging**